src/components/entity/state-info.js

```diff
@@ -82,12 +82,13 @@ class StateInfo extends PolymerElement {
       },
       hass: Object,
       stateObj: Object,
-      inDialog: Boolean
+      inDialog: Boolean,
+      config: Object
```

Comment:
This is wrong. These are components that are not part of lovelace and thus should not be made aware of lovelace config

Reply:
We should just have a title property that will be used if set.

Although I think name is better here. We override the state name that is being calculated after all.

Reply:
What do you think about creating a new stateObj with our custom name as friendly_name before we pass it to the state-cards?